SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+The following versions of onyx are currently being supported with security updates.
```

Comment:
If we do it like this we should add a section for each individuial package that we publish.
But as we are currently in the alpha stage for all package we might just want to tell people that we are in development with everything and update this file on the version 1.0.0 release.

Reply:
Changed the text, is it ok like this? :)